__tests__/universal/routes.spec.jsx

```diff
@@ -44,15 +47,24 @@ describe('routes', () => {
 
   beforeEach(() => jest.clearAllMocks());
 
-  it('should set up the root module route first', () => {
+  it('createRoutes should return array of length 2', () => {
+    mockHasChildRoutesValue = true;
```

Comment:
Best practice would be to utilize jest's mock functions instead, something like:

```suggestion
    hasChildRoutes.mockReturnValueOnce(true);
```

Reply:
Agreed and updated test case.